ServerCore/Pages/Submissions/AuthorIndex.cshtml

```diff
@@ -1,35 +1,66 @@
-﻿@page "/{eventId}/{eventRole}/AuthorSubmissions/{puzzleId}"
+﻿@page "/{eventId}/{eventRole}/AuthorIndex/{puzzleId?}"
 @model ServerCore.Pages.Submissions.AuthorIndexModel
 @{
     ViewData["Title"] = "AuthorIndex";
     ViewData["AdminRoute"] = "../Submissions/AuthorIndex";
     ViewData["AuthorRoute"] = "../Submissions/AuthorIndex";
     // TODO: Needs to handle implicit teams - ViewData["PlayRoute"] = "../Submissions/Index";
     ViewData["RoutingPuzzleId"] = ViewContext.RouteData.Values["puzzleId"];
-    Layout = "../Puzzles/_puzzleManagementLayout.cshtml";
+
+    if (Model.Puzzle != null)
+    {
+        Layout = "../Puzzles/_puzzleManagementLayout.cshtml";
+    }
+    else if (Model.Team != null)
+    {
+        Layout = "../Teams/_teamLayout.cshtml";
+    }
+    // TODO: Do we need a combined layout, or is that rare enough as to be unnecessary? Maybe those bar layouts should be sections/components?
```

Comment:
This is the state when you have selected both a puzzle and a team. From there, are you looking for more commands on the same puzzle? More commands for the same team? Both? I think the best answer is "Both", but currently there's no way to get commands for both due to the design.

I'm going to check this in since it's a big leap forwards, and create an issue to rethink how the layouts are chosen overall.